scripts/get_centerline_from_segmentation.sh

```diff
@@ -0,0 +1,119 @@
+
+# Usage:
+#     sct_run_batch -c config.json
+
+# The following global variables are retrieved from the caller sct_run_batch
+# but could be overwritten by uncommenting the lines below:
+# PATH_DATA_PROCESSED="~/data_processed"
+# PATH_RESULTS="~/results"
+# PATH_LOG="~/log"
+# PATH_QC="~/qc"
+#
+# Author: Jan Valosek
+#
+
+# Uncomment for full verbose
+set -x
+
+# Immediately exit if error
+set -e -o pipefail
+
+# Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
+trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
+
+# Print retrieved variables from sct_run_batch to the log (to allow easier debug)
+echo "Retrieved variables from from the caller sct_run_batch:"
+echo "PATH_DATA: ${PATH_DATA}"
+echo "PATH_DATA_PROCESSED: ${PATH_DATA_PROCESSED}"
+echo "PATH_RESULTS: ${PATH_RESULTS}"
+echo "PATH_LOG: ${PATH_LOG}"
+echo "PATH_QC: ${PATH_QC}"
+
+# Retrieve input params and other params
+SUBJECT=$1
+
+# get starting time:
+start=`date +%s`
+
+# SCRIPT STARTS HERE
+# ==============================================================================
+# Display useful info for the log, such as SCT version, RAM and CPU cores available
+sct_check_dependencies -short
+
+# Go to folder where data will be copied and processed
+cd $PATH_DATA_PROCESSED
+
+# Copy BIDS-required files to processed data folder (e.g. list of participants)
+if [[ ! -f "participants.tsv" ]]; then
+  rsync -avzh $PATH_DATA/participants.tsv .
+fi
+# Copy list of participants in results folder 
+if [[ ! -f "participants.json" ]]; then
+  rsync -avzh $PATH_DATA/participants.json .
+fi
+if [[ ! -f "dataset_description.json" ]]; then
+  rsync -avzh $PATH_DATA/dataset_description.json .
+fi
+
+# Copy source images
+# Note: we use '/./' in order to include the sub-folder 'ses-0X'
+rsync -Ravzh $PATH_DATA/./$SUBJECT .
+# Go to subject folder for source images
+cd ${SUBJECT}/anat
+
+# Define variables
+# We do a substitution '/' --> '_' in case there is a subfolder 'ses-0X/'
+file="${SUBJECT//[\/]/_}"
+
+# -------------------------------------------------------------------------
+# T2w
+# -------------------------------------------------------------------------
+# Add suffix corresponding to contrast
+file_t2w=${file}_T2w
+# Construct path to manually corrected SC segmentation
+FILESEG="${file_t2w}_seg"
+FILESEGMANUAL="${PATH_DATA}/derivatives/labels/${SUBJECT}/anat/${FILESEG}-manual.nii.gz"
+
+# Check if T2w SC seg exists
+if [[ -f ${FILESEGMANUAL} ]];then
+
+    # Copy manually corrected segmentation ground truth (GT) from derivatives folder (to do not alter files in derivatives)
+    # Note, this ground truth (GT) segmentation was obtained from reoriented and resampled images (this is why we are applying the same preprocessing steps also below)
+    rsync -avzh ${FILESEGMANUAL} ${FILESEG}.nii.gz
+    # Fit a regularized centerline on an already-existing cord segmentation.
+    sct_get_centerline -i ${FILESEG}.nii.gz -method fitseg -qc ${PATH_QC} -qc-subject ${SUBJECT}
```

Comment:
I would suggest to make these parameters explicit (eg in case the default value changes in SCT):
```
  -centerline-algo {polyfit,bspline,linear,nurbs}
                        Algorithm for centerline fitting. Only relevant with -method fitseg (default: bspline)
  -centerline-smooth <int>
                        Degree of smoothing for centerline fitting. Only for -centerline-algo {bspline, linear}. (default: 30)
```

Reply:
Thanks for the suggestion! Made explicit in https://github.com/ivadomed/model_sc_centerline/pull/2/commits/70f6e189fde93070e917bd96175266bf7c426d14